src/public/index.html

```diff
@@ -16,107 +16,129 @@
 </head>
 
 <body>
-    <div class="header">
-        <div class="title">CSV Import Plugin</div>
+<div class="header">
+    <div class="title">CSV Import Plugin</div>
+</div>
+<div id="cardsContainer">
+    <div id="homeContainer" class="card">
+        <div class="content">
+            <div class="titleCard">Choose a file to import</div>
+            <div class=uploadContainer>
+                <label for="importFile" class="fileUpload">Browse file</label>
+                <input type="file" id="importFile" name="importFile"><br>
+                <p id="fileName" class="fileName "></p>
+                <div id="fileError"></div>
+            </div>
+        </div>
+        <button id="readButton" class="primaryButton">Upload</button>
+        <button id="showExampleButton" class="link-button">
+            How to structure my file
+        </button>
     </div>
-    <div id="cardsContainer">
-        <div id="homeContainer">
-            <div class="content">
-                <div class="titleCard">Choose a file to import</div>
-                <div class=uploadContainer>
-                    <label for="importFile" class="fileUpload">Browse file</label>
-                    <input type="file" id="importFile" name="importFile"><br>
-                    <p id="fileName" class="fileName "> </p>
-                    <div id="fileError"></div>
-                </div>
+    <div id="fileStructureExampleContainer" class="card">
```

Comment:
because of the file format it is more complicated to review.
The changes start at the line 38 and end 57